src/bpftrace.h

```diff
@@ -282,6 +285,8 @@ class BPFtrace
   std::vector<uint8_t> find_empty_key(IMap &map, size_t size) const;
   bool has_iter_ = false;
   int epollfd_ = -1;
+  int debug_epollfd_ = -1;
+  char debug_output_buffer_[256 + 1];
```

Comment:
Nit: maybe keep this as a local var in the function it's used? IIUC stack vars are basically free. But keeping it as a member makes class instance bigger